### PR TITLE
Update glam extract query to transpose percentiles and histograms

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: sql/telemetry_derived/glam_client_probe_counts_extract_v1/query.sql [hunk 1]
```diff
@@ -1,49 +1,37 @@
...
-        AND total_users > 1000
+CREATE TEMP FUNCTION udf_js_flatten(
+    histogram ARRAY<STRUCT<key STRING, value FLOAT64>>
 )
+RETURNS STRING
+LANGUAGE js
+AS
+'''
+    let data = histogram.map(function(r) {
+        var obj = {};
```
Comment (emtwo) [inline sql/telemetry_derived/glam_client_probe_counts_extract_v1/query.sql:9]: nit: `let obj = {}`

File: sql/telemetry_derived/glam_client_probe_counts_extract_v1/query.sql [hunk 2]
```diff
@@ -1,49 +1,37 @@
...
 FROM
-    deduped
+    `moz-fx-data-shared-prod.telemetry.client_probe_counts`
 WHERE
-    rank = 1
+    channel = @channel
+    AND app_version IS NOT NULL
+    AND total_users > 1000
+GROUP BY
+    channel, app_version, app_build_id, os, metric, metric_type, key, process, client_agg_type, total_users
```
Comment (emtwo) [inline sql/telemetry_derived/glam_client_probe_counts_extract_v1/query.sql:37]: nit: newline

File: sql/telemetry_derived/glam_client_probe_counts_extract_v1/query.sql [hunk 3]
```diff
@@ -1,49 +1,37 @@
...
+    process,
     metric,
     -- BigQuery has some null unicode characters which Postgresql doesn't like, so we remove those here.
     -- Also limit string length to 200 to match column length.
     SUBSTR(REPLACE(key, r"\x00", ""), 0, 200) AS key,
     client_agg_type,
     metric_type,
     total_users,
-    TO_JSON_STRING(aggregates) AS aggregates
+    MAX(IF(agg_type="histogram", udf_js_flatten(aggregates), NULL)) AS histogram,
```
Comment (emtwo) [inline sql/telemetry_derived/glam_client_probe_counts_extract_v1/query.sql:28]: I assume the `MAX()` is so we return the non-null value which is the correct value for `histogram`? Maybe just a comment so it's clear.

Comment (robhudson) [reply sql/telemetry_derived/glam_client_probe_counts_extract_v1/query.sql:28]: A comment would be good, I agree. I'm using this as if it's a `COALESCE` but b/c of the `GROUP BY` I need an aggregation function to include this column here. I'll make a note of it.

**Pull request merged**